tests/lib/bin/mountinfo-tool

```diff
@@ -419,27 +419,32 @@ def renumber_ns(entry, seen):
         entry.root_dir = "{}:[{}]".format(ns_type, alloc_n(ns_type, ns_id))
 
 
-def rewrite_renumber(entries):
-    # type: (List[MountInfoEntry]) -> None
+class RewriteState(object):
+    """RewriteState holds state used in rewriting mount entries."""
+
+    def __init__(self):
+        # type: () -> None
+        self.seen_opt_fields = {}  # type: Dict[int, int]
+        self.seen_loops = {}  # type: Dict[int, int]
+        self.seen_snap_revs = {}  # type: Dict[Tuple[Text, Text], int]
+        self.seen_mount_ids = {}  # type: Dict[int, int]
+        self.seen_devices = {}  # type: Dict[Device, Device]
+
+
+def rewrite_renumber(entries, rs):
```

Comment:
Any reason for not making all the `rewrite_*` functions into methods of RewriteState class?

Reply:
They could either be methods on the mount entry or on the rewrite state. I think they are just functions that take two arguments. No strong feeling either way though.

Reply:
Ok, I'd probably make them methods, but no hard opinion and not a blocker.